WireAPI/Sources/WireAPI/Models/UserClient/MLSPublicKeys.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// A container of MLS public keys.
+
+public struct MLSPublicKeys: Equatable, Codable {
+
+    /// The key for the ed25519 ciphersuite.
```

Comment:
`ed25519` is not a ciphersuite itself but instead a signature component that can be used in ciphersuite. Same applies for the properties.

```suggestion
    /// The ed25519 signature key.
```